make/dev.mk

```diff
@@ -1,31 +1,42 @@
 DEV_MEMBER_NS := toolchain-member-operator
 DEV_HOST_NS := toolchain-host-operator
-DEV_REGISTRATION_SERVICE_NS := $(HOST_NS)
+DEV_REGISTRATION_SERVICE_NS := $(DEV_HOST_NS)
+DEV_ENVIRONMENT := dev
 
 .PHONY: dev-deploy-e2e
-dev-deploy-e2e:
+dev-deploy-e2e: deploy-e2e-to-dev-namespaces print-reg-service-link
+
+.PHONY: deploy-e2e-to-dev-namespaces
+deploy-e2e-to-dev-namespaces:
 	$(MAKE) deploy-e2e MEMBER_NS=${DEV_MEMBER_NS} HOST_NS=${DEV_HOST_NS} REGISTRATION_SERVICE_NS=${DEV_REGISTRATION_SERVICE_NS}
-	@echo "Deployment complete!"
-	@echo "To clean the cluster run 'make clean-e2e-resources'"
```

Comment:
I guess that instead of having two targets you could have:
```
.PHONY: dev-deploy-e2e
dev-deploy-e2e: MEMBER_NS=${DEV_MEMBER_NS}
dev-deploy-e2e: HOST_NS=${DEV_HOST_NS}
dev-deploy-e2e: REGISTRATION_SERVICE_NS=${DEV_REGISTRATION_SERVICE_NS}
dev-deploy-e2e: deploy-e2e print-reg-service-link
```
and the same for the local targets. But it's just a detail - your version is fine too ;-)

Reply:
Yeah.. It would look better! Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/44/commits/f72300737898a371e104da3c6d5e77f0347e0e14
Thanks.